docs/examples.md

```diff
@@ -348,90 +348,110 @@ const hmac = crypto
 
 ```php
 <?php
+// Use Guzzle HTTP client v6 installed with Composer https://github.com/guzzle/guzzle/
```

Comment:
For clarity I would mention that:

```
We recommend using guzzle through composer as default http client for php because it has well documented and nice api.
You can use any http library to connect into Checkout API.
If you can't install composer packages alternatively you can use: http://php.net/manual/en/book.curl.php
```